neuron-deploy/sm_notebook.rst

```diff
@@ -0,0 +1,16 @@
+.. _sagemaker-notebook:
+
+Sagemaker Notebook
+==================
+Introduction
+------------
+Use of Neuron with Sagemaker Notebook can be simple to achieve by following this
+    - PytorchNotebook_
+    - TensorflowNotebook_
+
+.. _PytorchNotebook: https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker_neo_compilation_jobs/deploy_pytorch_model_on_Inf1_instance
+.. _TensorflowNotebook: https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker_neo_compilation_jobs/deploy_tensorflow_model_on_Inf1_instance
+
+Debug Hints
+-----------
+Sagemaker endpoints have cloudwatch metrics. For more details please check `here <https://docs.aws.amazon.com/sagemaker/latest/dg/monitoring-cloudwatch.html>`_
```

Comment:
replace with "You can Monitor Amazon SageMaker with Amazon CloudWatch. For more details...."